dropwizard-jersey/src/main/java/io/dropwizard/jersey/DropwizardResourceConfig.java

```diff
@@ -177,8 +185,10 @@ String cleanUpPath(String path) {
         private final String rootPath;
         private final Class<?> klass;
 
-        EndpointLogger(String urlPattern, Class<?> klass) {
-            this.rootPath = urlPattern.endsWith("/*") ? urlPattern.substring(0, urlPattern.length() - 1) : urlPattern;
+        EndpointLogger(String contextPath, String urlPattern, Class<?> klass) {
+            final String rootPattern = urlPattern.endsWith("/*") ? urlPattern.substring(0, urlPattern.length() - 1) : urlPattern;
```

Comment:
Looks like by default the printed path starts with double slashes "//".
If urlPattern == '/*', then rootPattern = '/', and if contextPath == '/', rootPath = '//'

Reply:
Thanks for reporting this! I've created #2279 to fix the output.